app/javascript/components/settings/theme-preference-form/InfoMessage.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+export function InfoMessage({
+  insidersStatus,
+  insidersPath,
+}: {
+  insidersStatus: string
+  insidersPath: string
+}): JSX.Element {
+  switch (insidersStatus) {
+    case 'active':
+    case 'active_lifetime':
+      return (
+        <p className="text-p-base mb-16">
+          We hope you enjoy it. Thanks for all your support.
+        </p>
+      )
+    case 'eligible':
+    case 'eligible_lifetime':
+      return (
+        <p className="text-p-base mb-16">
+          You&apos;re eligible to join Insiders.{' '}
+          <a href={insidersPath}>Get started here.</a>
+        </p>
+      )
```

Comment:
Maybe we have two separate messages here? As a lifetime insider is something slightly different and one could already be an insider.